call-home/src/bin/callhome/main.rs

```diff
@@ -29,6 +32,10 @@ struct CliArgs {
     /// The namespace we are supposed to operate in.
     #[arg(short, long, default_value = DEFAULT_NAMESPACE)]
     namespace: String,
+
+    /// The endpoint to fetch events stats.
+    #[clap(long, short, default_value = DEFAULT_TRANSMISSION_URL)]
+    transmission_url: Url,
```

Comment:
What does transmission mean?
should this be renamed to something more like `stats_aggregator` ?